package.json

```diff
@@ -1,82 +1,108 @@
+    "contributors": [ {
+            "name": "Dmitri Zagidulin",
+            "url": "https://github.com/dmitrizagidulin/"
+        },
+        {
+            "name": "Daniel Friedman",
+            "url": "https://github.com/dan-f/"
+        }
+    ],
+    "license": "MIT",
+    "repository": {
+        "type": "git",
+        "url": "git://github.com/linkeddata/rdflib.js.git"
+    },
+    "homepage": "http://github.com/linkeddata/rdflib.js",
+    "bugs": "http://github.com/linkeddata/rdflib.js/issues",
+    "dependencies": {
+        "async": "^0.9.x",
+        "jsonld": "^0.4.5",
+        "n3": "^0.4.1",
+        "xmldom": "^0.1.22",
+        "xmlhttprequest": "^1.7.0"
+    },
+    "devDependencies": {
+        "babel-cli": "^6.11.4",
+        "babel-preset-es2015": "^6.9.0",
+        "babel-register": "^6.18.0",
+        "babelify": "^7.3.0",
+        "browserify": "^13.0.0",
+        "chai": "^3.5.0",
+        "fs-grep": "^0.0.5",
+        "minifyify": "^7.3.2",
+        "mocha": "^3.2.0",
+        "node-mkdirp": "0.0.1",
+        "rimraf": "^2.6.1",
+        "jsdiff-cli": "^0.1.2"
+    },
+    "scripts": {
+        "test": "npm run test:unit && npm run test:serialize",
+        "test:unit": "mocha --growl --require babel-register tests/unit/**-test.js",
+        "test:unit:dev": "mocha --watch --growl --require babel-register tests/unit/**-test.js",
+        "test:serialize": "npm run lib  && npm run serialize-tests && npm run cleantest",
+        "dist": "mkdirp dist",
+        "dist/rdflib.js": "browserify -r ./src/index.js:rdflib --exclude 'xmlhttprequest' --standalone '$rdf' -t [ babelify --presets [ es2015 ] ] -o dist/rdflib.js",
+        "dist/rdflib.min.js": "browserify -r ./src/index.js:rdflib --exclude 'xmlhttprequest' --standalone '$$rdf' -d -t [ babelify --presets [ es2015 ] ] -p [ minifyify --no-map --uglify [ --compress [ --dead_code --conditionals --unused --if_return ] --mangle --screw-ie8 ] ] -o dist/rdflib.min.js",
+        "all": "npm run dist && npm run dist/rdflib.js && npm run dist/rdflib.min.js",
+        "lib": "babel src -d lib",
+        "detach": "git checkout origin/master &&  git reset--hard HEAD",
+        "clean": "rimraf dist/* lib/*",
+        "cleantest": "rimraf tests/serialize/,*",
+        "status": "@git branch -v && @git status -s",
+        "minify": "npm run dist/rdflib.min.js",
+        "size": "node-line-counter .",
+        "gh-pages": "npm run detach && git branch - D gh - pages ||: && git checkout - b gh - pages && npm run all && git add - f dist /*.js src/*.js *.js && git commit -m 'gh-pages: update to latest' && git push -f origin gh-pages && git checkout master",
+        "prepublish": "npm run lib",
+        "serialize-tests": "npm run t1 && npm run t2 && npm run t3 && npm run t4 && npm run t5 && npm run t6 && npm run t7",
+        "t1": "cd tests/serialize && node ./data.js -in=t1.ttl -format=application/rdf+xml -out=,t1.xml && fs-grep http://www.w3.org/2001/XMLSchema#integer ,t1.xml",
```

Comment:
I think these would be a bit clearer (short of a refactor of the tests themselves) if renamed to something along the lines of `serialize-test-1`, `serialize-test-2`, etc.